openedx/core/djangoapps/content/search/tests/test_documents.py

```diff
@@ -295,10 +299,16 @@ def test_html_library_block(self):
         }
 
     def test_collection_with_library(self):
-        doc = searchable_doc_for_collection(self.collection)
+        doc = {}
+        doc.update(searchable_doc_for_collection(self.collection))
```

Comment:
The existing line `doc = searchable_doc_for_collection(self.collection)` is more concise and efficient than two lines:

```
        doc = {}
        doc.update(searchable_doc_for_collection(self.collection))
```
So I'm not really sure why you changed this?

Reply:
Updated: https://github.com/openedx/edx-platform/pull/35483/commits/199d1eda887537d33e88bed73829b82b312fc278